packages/core/custom-elements-lsp/jest/custom-elements.ts

```diff
@@ -13,7 +13,7 @@ let manifest: string;
  * Builds a stubbed `CustomElementsResource` and constructs a `CustomElementsServiceImpl`.
```

Comment:
All of the changes in `core/custom-elements-lsp` are just linting changes arising from needing the bump the eslint prettier plugin